cluster/storage/src/main/java/com/linkedin/openhouse/cluster/storage/selector/DefaultStorageSelector.java

```diff
@@ -0,0 +1,27 @@
+package com.linkedin.openhouse.cluster.storage.selector;
+
+import com.linkedin.openhouse.cluster.storage.Storage;
+import com.linkedin.openhouse.cluster.storage.StorageManager;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+
+/**
+ * An implementation of {@link StorageSelector} that returns cluster level storage for all tables
```

Comment:
nit: `that returns storage thats marked as default-type for the cluster in yaml configuration`